controller/xt_ha/impl.go

```diff
@@ -40,33 +40,33 @@ func (f factory) GetStrategyName() string {
 }
 
 func (f factory) NewStrategy() xt.Strategy {
```

Comment:
do the same for `factory` methods?

Reply:
Not strictly necessary, since the factory carries no state, but I made it consistent along with the rest of the strategies.